end2end-tests/models-jackson/src/test/resources/deep_nested_polymorphic/third_level_child11.json

```diff
@@ -0,0 +1,16 @@
+{
+  "rootField1" : "root field 1",
+  "rootField2" : false,
+  "firstLevelField1" : "first level field 1",
+  "firstLevelField2" : null,
+  "metadata" : {
+    "obj" : {
+      "filed1" : "field 1",
+      "field2" : "field 2"
+    }
+  },
+  "creationDate" : 1,
+  "secondLevelDiscriminator" : "thirdLevelChild1",
+  "rootDiscriminator" : "firstLevelChild",
+  "firstLevelDiscriminator" : "secondLevelChild1"
```

Comment:
Why are  3 different levels of discriminator all at the root level in this json?

Reply:
That is one to one what is generated during the serialization. In case there isn't something wrong with the setup that is.